ritasamir/assignment1/Q_01.java

```diff
@@ -0,0 +1,37 @@
+package assignment_1;
+
+import java.util.Arrays;
+
+public class Q_01 {
+
+	public boolean checkAnagram(String firstStr, String secondStr) {

```

Comment:
Very nice start! :)
I see here you wrote a method that would compare two sentences. Please add a comment above (using a block comment/javadoc, more info [here](http://drjava.org/docs/user/ch10.html)) explaining the input for this method.